shark/examples/shark_inference/minilm_load_benchmark_tf.py

```diff
@@ -0,0 +1,41 @@
+import tensorflow as tf
+from transformers import BertModel, BertTokenizer, TFBertModel
+from shark.shark_inference import SharkInference
+from shark.shark_importer import shark_load
+from shark.parser import parser
+import os
+
+gpus = tf.config.experimental.list_physical_devices('GPU')
+for gpu in gpus:
+  tf.config.experimental.set_memory_growth(gpu, True)
+
+parser.add_argument(
+    "--download_mlir_path",
+    type=str,
+    default="minilm_tf_inference.mlir",
```

Comment:
something like `microsoft/MiniLM-L12-H384-uncased_inference`, we can later add the .mlir extension. This makes sure that user can navigate to and fro shark_hub and hugging face to find the right model.